doc-coverage/raco.rkt

```diff
@@ -0,0 +1,75 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+(module+ main
+
+  (define binding (make-parameter #f))
+  (define ratio (make-parameter #f))
+  (define ignore (make-parameter #f))
+  (define error-on-exit? (make-parameter #f))
+
+  (define args
+    (command-line
+     #:program (short-program+command-name)
+     #:once-any
+     [("-b" "--binding") b
+      "Check the documentation for a specific binding"
+      (binding (string->symbol b))]
+     [("-r" "--ratio") r
+      "Specify required documentation ratio"
+      (ratio (string->number r))]
+     [("-s" "--skip") s
+      "Specify regex of bindings to ignore"
+      (ignore (regexp s))]
+     #:args (file . files)
+     (cons file files)))
+
+  (for ([a (in-list args)])
+    (let/ec break
+
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (set! a (string->symbol a)))])
```

Comment:
I find there's a sizable difference between a linear program written as "increment x, mutate y, write foo, set error flag" and a linear program written as "bring module into scope, branch on choice of check, run check with given files" where that information is in function names. It's very easy for the author of code to identify which blocks have which meanings, but as a separate reader it's difficult for me to figure that out. More well-named helpers would be good here I think.

As for how to actually make this particular thing work, it does seem tricky. The best idea I've got so far is a helper macro:

```racket
(define-syntax-rule (when-fail expr body ...)
  (with-handlers ([exn:fail? (lambda (_) body ...)]) expr))

(define (try-namespace-require a)
  (define (namespace-require/return a)
    (namespace-require a)
    a)
  (when-fail (namespace-require/return a)
    (when-fail (namespace-require/return (string->symbol a))
      #f))
```

It's especially tricky since the rest of the loop body needs to know whether the `a` used was a symbol or string.